.github/workflows/buildExecuteCustom.yaml

```diff
@@ -60,12 +60,20 @@ jobs:
           lastRC="$(./scripts/lastVersionRC.sc)"
           lastStable=$(./scripts/lastVersionStable.sc)
 
-          ./scripts/raport-regressions.scala $scalaVersion > raport-full.md
-          ./scripts/raport-regressions.scala $scalaVersion --compareWith=$lastRC > raport-compare-$lastRC.md
-          ./scripts/raport-regressions.scala $scalaVersion --compareWith=$lastStable > raport-compare-$lastStable.md
+          scala-cli scripts/raport-regressions.scala scripts/md_printer.scala -- $scalaVersion > raport-full.md
+          scala-cli scripts/raport-regressions.scala scripts/md_printer.scala -- $scalaVersion --compareWith=$lastRC > raport-compare-$lastRC.md
+          scala-cli scripts/raport-regressions.scala scripts/md_printer.scala -- $scalaVersion --compareWith=$lastStable > raport-compare-$lastStable.md
 
       - name: Upload raports
         uses: actions/upload-artifact@v3
         with:
           name: build-raports
           path: ${{ github.workspace }}/raport-*.md
+
+      - name: Push raport
+        uses: ./.github/actions/push-raport-to-gh-page
```

Comment:
Can we add an additional boolean workflow parameter controlling whether should we push the report? 
That way we would be able to still post results of builds for RC/stable versions, but we would be able also to skip publishing results from testing custom compiler branches (from compiler PRs). If so, it should not push results by default (needs explicit confirmation)